index.js

```diff
@@ -160,8 +165,8 @@ function getContentType(limit, skip, contentSettings, itemsPulled) {
                 frontMatter.createdAt = item.sys.createdAt;
                 frontMatter.date = item.sys.createdAt;
                 for (let field of Object.keys(item.fields)) {
-                    if (field === contentSettings.mainContent) {
-                        // skips to prevent duplicating mainContent in frontmatter
+                    if (field === contentSettings.mainContent || field === contentSettings.fileNameField) {
```

Comment:
I'm debating whether or not we should skipping adding the fileNameField to the frontmatter. My thought is "What if someone has a dual purpose for a field?"

For example what if somebody wanted to generated a filename based on a date / time field? Or what if they wanted to generated a file based on the Title? They would no longer have access to it in the front-matter.

So I'm leaning towards keeping the field in the frontmatter even though it's a bit of data duplication.